industrial_robot_simulator/industrial_robot_simulator

```diff
@@ -270,7 +271,11 @@ class IndustrialRobotSimulatorNode():
         # retrieve update rate
         motion_update_rate = rospy.get_param('motion_update_rate', 100.);  #set param to 0 to ignore interpolated motion
         self.motion_ctrl = MotionControllerSimulator(num_joints, initial_joint_state, update_rate=motion_update_rate)
-
+        
+        # React to published e-stop
+        self.e_stop_state = False
+        self.e_stop_sub = rospy.Subscriber('fake_e_stop', Bool, self.e_stop_changed_callback)
```

Comment:
Would it be better to *not* call this `fake_e_stop`? The whole script is already a simulator/simulation of a real robot, and those typically don't have `fake_e_stop` topics.

If we want to add this here, I would propose the rename the subscribed topic. If the producing side would like to call the topic `fake_e_stop`, remapping could be used.